docs/contributors/advanced/review-a-merge-proposal.md

```diff
@@ -1,240 +1,171 @@
 
 
 ## What to review
 
-The following is an overview of common things to look out for. Obviously, each
-case is different and a reviewer can also check for anything else they consider
-appropriate for the situation.
+The following is an overview of common things to check.
 
 
-### Check the new changelog stanza
+### New changelog stanza
 
-* Contains a bug pointer like `(LP: #12345678)` to all associated cases.
+* Contains a bug pointer in the format `LP: #12345678` for all associated cases.
 
-* Correctly formatted entries according to the
-  [changelog policy](https://www.debian.org/doc/debian-policy/ch-source.html#debian-changelog-debian-changelog)
-  and {ref}`hints for merges <merge-fix-the-changelog>`.
+* Correctly formatted entries according to the [changelog policy](https://www.debian.org/doc/debian-policy/ch-source.html#debian-changelog-debian-changelog) and {ref}`hints for merges <merge-fix-the-changelog>`.
 
-* Lists all changes made -- the changelog shall be a complete representation.
+* Lists all changes made -- the changelog is a complete representation.
 
 * Proper version -- check against {ref}`version-strings`.
 
-* Proper release -- `dch` or habits could have selected the wrong one.
-
-* Proper author and email.
+* Proper release -- often incorrect due to the {command}`dch` tool or a human error.
 
+* Author name and email address.
 
-### Ensure Documentation/Release Notes are updated
 
-* We always queue up things that eventually need to be mentioned in the
-  Release Notes. This check in the MP review is a reminder about that.
+### Documentation and release notes updates
 
-* If the update/merge has implications that need to be documented, documentation
-  should always be updated alongside the change. Sometimes things get in via a
-  sync or upstream changes -- as you spot these, make sure to spawn a
-  proper backlog tracker for them.
+* All significant developments that need mentioning in Ubuntu release notes should be queued in the document.
 
+* If the update/merge has implications that need to be documented, ensure documentation is updated alongside the change. When things that require documentation arrive via a sync or upstream changes, create a backlog tracker for them.
 
-### Check for indirect changes
 
-* Some changes can imply that the packaging needs to be adapted. Check content
-  and release notes if there are any changes like that.
+### Indirect changes
 
-* When merging from Debian or Upstream it is worth checking if there are even
-  newer versions that would be worth incorporating.
+* Some changes imply that the packaging needs to be adapted. Check the content and release notes for any changes like that.
 
-  * Bad things happen -- check upstream to see if a release has been withdrawn
-    or needs an immediate fix-up due to unintentional breakage.
+* When merging from Debian or upstream:
+  * Check if there are newer versions to incorporate.
```

Comment:
I feel like this change has altered the implied meaning to be that "one should incorporate newer versions (always)", vs the original which was more like "there might be newer versions but they may or may not be worth incorporating now". Probably worth clarifying which side of the fence we're supposed to be on.